### ISSUE TITLE
Allow option to reset to page 1 on filter/s

### ISSUE THREAD

**Issue by joshuacura** [ssuperczynski/ngx-easy-table#191]
Current behavior seems it sticks to the current active page, and just adjusts to the count of the filtered items - that is, if the current page is still valid for the filtered results, it will display that page, else it will show as last page when the current page does not fit the count.

For server pagination, there may be use cases where we need to always display page 1 for filtering (even on clearing filters as well). May we request your help to implement this feature? 

Thanks!

**Comment by ssuperczynski** [comment]
Hey @joshuacura, could you describe step by step where is the error and how proper solution should look like.

**Comment by joshuacura** [comment]
The scenario is, for example we have multiple pages (1-5), then the current page is on page 3. Then I applied a filter text, so rows will be filtered... if the total rows can be paginated on 3 or more ( >= 3) pages, then the view will still show in page 3; else, it will show page 1 (still filtered) because there is no page 3. 

So if it's possible to set the current page to first page when applying filters with pagination, and when we clear the filter, we also set it again to first page, always. If this could be applied a configuration setting, then I think this would just be an additional feature then.

Thanks!
